client/style.css

```diff
@@ -0,0 +1,43 @@
+* {
```

Comment:
is the * means the root:: ? I haven't seen this before, interesting.

Reply:
Just means styling of all elements I believe